insights/parsers/haproxy_cfg.py

```diff
@@ -81,6 +90,17 @@ def _parse_content(content):
 
 @parser(Specs.haproxy_cfg)
 class HaproxyCfg(Parser):
-    """Class to parse file ``haproxy.cfg``."""
+    """
```

Comment:
@jobselko Since both parsers use the _parse_content function. Could you change the _parse_content function to a base class with a parse_content  function that both parsers inherit, that way the parsers can just have pass under them similar to how the ps parser is setup? In addition would it be possible to move the doc string under the new base class?

Reply:
@ryan-blakley Updated. Let me know if I missed anything.

Reply:
@jobselko That looks good, just one thing I see, could you add a pass under the comments on each of the parser classes like the below?

```python
@parser(Specs.haproxy_cfg)
class HaproxyCfg(HaproxyFile):
    """
    Class to parse file ``/var/lib/config-data/puppet-generated/haproxy/etc/haproxy/haproxy.cfg`` or ``haproxy.cfg``.
    """
    pass
```

Reply:
@ryan-blakley According to the pylint, `pass` is not needed, but I will add it.

Reply:
@jobselko Thank you, everything looks good to me.